parts/linux/cloud-init/artifacts/components.json

```diff
@@ -382,6 +382,29 @@
       "multiArchVersions": [
         "v0.7.1"
       ]
+    },
+    {
+      "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:*",
+      "amd64OnlyVersions": [],
+      "multiArchVersions": [
```

Comment:
that's way too may version. after we review the Schema, we will need to keep 3 images only (one per AKS versions)

Reply:
I believe we'll need to keep corresponding kube-proxy version for each unique major.minor.patch k8s version